packs/osx-attacks.conf

```diff
@@ -122,12 +122,18 @@
       "description" : "(http://www.thesafemac.com/osxfkcodec-a-in-action/)",
       "value" : "Artifact used by this malware"
     },
-    "iWorm": {
+    "iWorm_1": {
```

Comment:
Don't change this, or it will invalidate/duplication all the existing `iWorm` detections, haha.